src/Paystack.php

```diff
@@ -98,59 +98,39 @@ private function setRequestOptions()
 
     /**
      * Initiate a payment request to Paystack
-     * @return Unicodeveloper\Paystack\Paystack
      */
     public function makePaymentRequest()
     {
-        $this->setResponse('/transaction/initialize');
-
-        return $this;
-    }
-
-    /**
-     * Make the client request and get the response
-     * @param string $relativeUrl
-     * @return Unicodeveloper\Paystack\Paystack
-     */
-    public function setResponse($relativeUrl)
-    {
         $data = [
             "amount" => intval(request()->amount),
             "reference" => request()->reference,
             "email" => request()->email
         ];
```

Comment:
Why are you merging `makePaymentRequest` and `setResponse` method together? 

Please don't merge them together